content/docs/getting_started/tutorials/dataform-101.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Dataform 101
+priority: 0
+---
+
+This tutorial demonstrates the first few steps in configuring a new project.
+
+Google Cloud is used as the data warehouse in this example, but [other warehouses](../how_to_guides/set-up-warehouse) are viable too.
+
+## Log in or sign up
+
+After opening [the app](https://app.dataform.co/).
+
+<img src="/static/images/how_to_guides/dataform_101/1_sign_up.png" style="width: 100%;" />
+
+If not signing in with GitHub, Google or Microsoft then you will need to verify your email address.
+
+## Create new project
+
+This will create a new, empty project.
+
+<img src="/static/images/how_to_guides/dataform_101/2_new_project.png" style="width: 100%;" />
+
+## Give the project an inspirational name
+
+<img src="/static/images/how_to_guides/dataform_101/3_project_name.png" style="width: 100%;" />
+
+## Choose to "create a table" from the file templates
+
+Ignore the instruction to the configure the data warehouse; we'll do that next.
+
+<img src="/static/images/how_to_guides/dataform_101/4_file_templates.png" style="width: 100%;" />
+
+## Connect to your warehouse
+
+Big green button towards towards the central right.
+
+<img src="/static/images/how_to_guides/dataform_101/5_create_table.png" style="width: 100%;" />
```

Comment:
Nailed it
![Screenshot 2019-12-27 at 11 13 30](https://user-images.githubusercontent.com/13434377/71515182-fa241f00-2899-11ea-960d-c37cf14a425b.png)

Reply:
Fixed, added a bit of padding to images too as the extra whitespace is nice

Reply:
Hah, so the the solution to this would be to put the rule in `documentation.css` and scope it under the mainContent class like:
```css
.mainContent img {
  width: 100%;
}
```

Reply:
Ah yes that's the syntax, thanks.